react/features/recording/components/Recording/web/StartRecordingDialogContent.tsx

```diff
@@ -37,9 +37,11 @@ class StartRecordingDialogContent extends AbstractStartRecordingDialogContent {
      * @returns {React$Component}
      */
     render() {
+        const _renderRecording = this.props._isModerator || this.props._hasRecordingFeature;
```

Comment:
Similar as above, it should be based on the content of features, with the moderator flag only being used to infer defaults